.travis.yml

```diff
@@ -19,22 +19,19 @@ env:
     - MODE=saucelabs
 
 stages:
-  - lint
+  - check
   - test
-  - bundlesize
-  - docs
 
 jobs:
+  allow_failures:
```

Comment:
Since you are already touching this, Please disable the bundle size check on master build. It makes only sense on Pull requests and raises red flag before merging to master.

Reply:
With `allow_failures` it won't make the whole test to fail only one job will fail status will be green including in master.

Reply:
Yeah i understand, But my point is to not even run the bundle size check on master since it does not help us in any way.

Reply:
We can do disable in another PR.

Reply:
I think it's ok to have it on master as well. Currently we can't make decisions regarding the bundle size per PR (so far we only increase the bundle limit). We should have periodic audits of the bundle and having this check on master gives us the current status of this check.